pom.xml

```diff
@@ -51,6 +51,14 @@
         <cucumber-expressions.version>10.1.0</cucumber-expressions.version>
         <datatable.version>3.3.1</datatable.version>
         <tag-expressions.version>2.0.4</tag-expressions.version>
+        <!--
+        IMPORTANT!
+        Whenever messages.version is updated, also update the files from the compatibility kit:
+        cd ../cucumber # the monorepo
+        git checkout messages/vMAJOR.MINOR.PATCH
+        cd ../cucumber-jvm
+        ./compatibility/copy-cck-expectations.sh
+        -->
         <messages.version>12.0.0</messages.version>
```

Comment:
This includes the new error messages. I'm assuming the parser doesn't know this and we don't have test for it.

Reply:
The parser does know about it. Fixed in cd2e019716f1f49fe260fe85e53f5d08a953ea66